tos/tos_jsonenums.go

```diff
@@ -0,0 +1,116 @@
+// generated by jsonenums -type=ToS; DO NOT EDIT
+
+package tos
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+var (
+	_ToSNameToValue = map[string]ToS{
```

Comment:
nit, suggest calling this `_tosNameToValue` and below `_tosValueToName` (the caps + underscore is a little confusing as to whether it's exported or not)